src/seedu/addressbook/data/person/Person.java

```diff
@@ -25,6 +28,7 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        this.sequenceNumber = ++nextSequenceNumber;
```

Comment:
use of prefix increment is a little misleading / redundant in a way. your very first sequence number is 1 which also implies that the initial value of 0 is not used at all.